DEPLOYMENT.md

```diff
@@ -2,24 +2,40 @@
 
 ## Encryption
 
-Lightning enforces encryption at rest for Credentials, for which an 
-encryption key must be provided when running in production.
+Lightning enforces encryption at rest for Credentials, for which an encryption
+key must be provided when running in production.
 
-The key is expected to be a randomized set of bytes, 32 long; and Base64
-encoded when setting the environment variable.
+The key is expected to be a randomized set of bytes, 32 long; and Base64 encoded
+when setting the environment variable.
 
-There is a mix task that can generate keys in the correct shape for use
-as an environment variable:
+There is a mix task that can generate keys in the correct shape for use as an
+environment variable:
```

Comment:
@taylordowns2000 as an aside, what are you ruler numbers in your editor? I use 80 chars, and so does Elias - no real issue here; just noticing some formatting that didn't change the actual copy.

Reply:
I _think_ 80, but I've actually been carrying around the `.prettierrc` for a wihle now and forgot to commit to main. Will add shortly. It shouldn't be an individual devs decision :-)

Reply:
it was `proseWrap` @stuartc ! i think that likely one of your vsCodes wasn't wrapping markdown files cause there's an extra option needed for prettier. have added a very basic `.prettierrc` now:
```
{ "proseWrap": "always", "printWidth": 80 }
```